plasma_framework/contracts/src/exits/payment/routers/PaymentInFlightExitRouterArgs.sol

```diff
@@ -70,4 +70,27 @@ library PaymentInFlightExitRouterArgs {
         bytes competingTxInclusionProof;
         bytes competingTxWitness;
     }
+
+    /*
+     * @notice Wraps arguments for challenge in-flight exit input spent.
+     * @param inFlightTx RLP encoded in-flight transaction.
+     * @param inFlightTxInputIndex Index of input that's been spent.
+     * @param spendingTx RLP encoded spending transaction.
+     * @param spendingTxInputIndex Index of spent input in spending transaction.
+     * @param spendingTxInputOutputType Output type of spent input.
+     * @param spendingTxWitness Witness for spending transaction.
+     */
+    struct ChallengeInputSpentArgs {
+        bytes inFlightTx;
+        uint8 inFlightTxInputIndex;
+        bytes spendingTx;
+        uint8 spendingTxType;
```

Comment:
We can get that from spending tx wire format. No need to provide it as argument.